### PR TITLE
Updated board guide

### PR DESCRIPTION
**Pull request by bvhme**

### CODE REVIEW

File: board-guide/readme.md [hunk 1]
```diff
@@ -1,46 +1,30 @@
...
+> *Goal of this guide:* Explain how to work with the achievable board in the interface and with achievables.
 
-# How to use the Achievable board
+# How to use the achievable board
 
-<img src="../images/board.png" width="700">
+The main tool in our process is the achievable board. The board is the place where we keep track of our ideas, where they are in their lifecycle and who is working on what.
 
-The Board is part of [The Interface](https://interface.newatoms.com/). Here is where we can see what needs to be done. We can add *cards* which always contain a *user story*. They're part of our Agile way of working.
+![The board in action](../images/board.png)
```
Comment (thomasdebeus) [inline board-guide/readme.md:7]: I think the image should be from the live branch (green theme) and maybe with the header for more friendlyness? 😁 Doesn't matter that much, but it's not 'in action' right?

File: board-guide/readme.md [hunk 2]
```diff
@@ -1,46 +1,30 @@
...
 
-* Goal (To)
-* Person (as a)
-* Action (I can)
+Through its life an achievable travels from left to right.
 
+The first phase is the ‘creation’ phase and the last one is the ‘live’ phase, which is when the achievable has been achieved, it is out there in the world, out of our hands, more than often for others to see.
 
-### Example:
+In every phase an achievable has offers kinds of actions you can perform on it. For instance in the ‘next’ phase you can assign an achievable to you and move it to ‘in progress’ while if you have an achievable assigned to you for review it enables you to mark it as merged by moving it to the ‘ready’ phase.
```
Comment (thomasdebeus) [inline board-guide/readme.md]: 'offers'?

File: board-guide/readme.md [hunk 3]
```diff
@@ -1,46 +1,30 @@
...
-## User Story
+## The life of an achievable
 
-A user story is a description consisting one sentence and is chopped up in three parts. It describes a task that has to be done. Never make big user stories, always try to cut a problem into the smallest user stories that still make sense.
+From creation to going live [achievables](../glossary/achievable.md) go through distinct phases, like growing up. On the board there is one column for every [phase](../glossary/phase.md) an achievable is in.
 
-* Goal (To)
-* Person (as a)
-* Action (I can)
+Through its life an achievable travels from left to right.
```
Comment (thomasdebeus) [inline board-guide/readme.md]: Maybe explaning that an achievable can be seen as a physical card to make it more tangible.

File: board-guide/readme.md (hunk 1 above)
Comment (bvhme) [inline board-guide/readme.md:7]: Yeah, only it's changed so much that it'll be unrecognisable after we merge this

**Pull request merged**